test/e2e/autoscale_test.go

```diff
@@ -477,3 +479,53 @@ func TestTargetBurstCapacityMinusOne(t *testing.T) {
 		t.Fatalf("Initial state never achieved: %v", err)
 	}
 }
+
+func TestFastScaleToZero(t *testing.T) {
+	t.Parallel()
+	cancel := logstream.Start(t)
+	defer cancel()
+
+	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency,
+		rtesting.WithConfigAnnotations(map[string]string{
+			autoscaling.TargetBurstCapacityKey: "-1",
+			autoscaling.WindowAnnotationKey:    autoscaling.WindowMin.String(),
```

Comment:
As discussed in chat earlier, I think it'd be nice if we didn't need this annotation to get the rapid scale to zero post-"Ready" during initial deployment.

Reply:
Note: I think this should be followed up separately, but removing this would make an interesting test of that functionality.

Reply:
Yes, this needs a change to the _service_ itself.
I'll try to do this Monday/Tuesday.